### PR TITLE
Refactor MVP and add org selection

### PR DESCRIPTION
**Pull request by oyelekci**
https://github.com/UKHomeOffice/application-container-platform-board/issues/2251

### CODE REVIEW

File: pkg/in/create.go [hunk 1]
```diff
@@ -0,0 +1,167 @@
+package in
+
+import (
+	"encoding/json"
+	"fmt"
```
Comment (vinn946) [inline pkg/in/create.go:5]: IMO using the log package might be useful in the future for things such as timestamp references. However perfectly fine for a first iteration

File: pkg/in/create.go [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+	}
+	defer res.Body.Close()
+
+	// read HTTP response
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return "", fmt.Errorf("could not read JSD response body %v", err)
+	}
+
+	// FIXME
```
Comment (vinn946) [inline pkg/in/create.go]: Forgotten comment perhaps?

File: pkg/in/update.go [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+		defer res.Body.Close()
+
+		// read HTTP response
+		_, err = ioutil.ReadAll(res.Body)
+		if err != nil {
+			return "", fmt.Errorf("could not read JSD response body %v", err)
+		}
+
+		// FIXME
+		//fmt.Printf("sent request, JSD replied with: %v", string(body))
```
Comment (vinn946) [inline pkg/in/update.go]: Likewise from above comment

File: pkg/in/update.go [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+		return fmt.Errorf("could not form JSD URL: %v", err)
+	}
+
+	// create client and request
+	c := &caller.Client{
+		BaseURL:    surl,
+		HTTPClient: &http.Client{Timeout: 5 * time.Second},
+	}
+
+	var t string
```
Comment (vinn946) [inline pkg/in/update.go:123]: Would be good to add a small comment, referencing what the `t` value is used for - without knowing the internal workings of the application, it's hard to know what this exactly does

**Review by vinn946 — APPROVED** [review]
LG as a first iteration 👍🏼  - A few nitpicking comments were made.
There are a few duplications of functionality between in/out packages which can definitely be tidied up, however knowing that it's been fully tested and working, these can be iterated over time.

File: pkg/in/create.go (hunk 1 above)
Comment (oyelekci) [reply pkg/in/create.go:5]: cloudwatch takes care of timestamps so avoiding log generally (one less package)

**Pull request merged**